plugins/woocommerce/tests/e2e-pw/tests/merchant/products/block-editor/create-simple-product-block-editor.spec.js

```diff
@@ -66,378 +66,365 @@ test.describe( 'General tab', { tag: tags.GUTENBERG }, () => {
+					.getByRole( 'button', {
+						name: 'Publish',
+					} )
+					.click();
+
+				await expect(
+					page.getByLabel( 'Dismiss this notice' )
+				).toContainText( 'Product published' );
+
+				const title = page.locator(
+					'.woocommerce-product-header__title'
+				);
+
+				// Save product ID
+				const productIdRegex = /product%2F(\d+)/;
+				const url = page.url();
+				const productIdMatch = productIdRegex.exec( url );
+				// This isn't really a conditional branch in the test;
+				// just making sure we don't blow up if the regex doesn't match
+				// (it will be caught in the expect below).
+				// eslint-disable-next-line playwright/no-conditional-in-test
+				productId = productIdMatch ? productIdMatch[ 1 ] : null;
+
+				expect( productId ).toBeDefined();
+				await expect( title ).toHaveText( productData.name );
+			} );
+
+			// Note for future refactoring: It would be good to reuse the verification step
+			// from product-create-simple.spec.js, as both tests are just verifying that the
+			// product was created correctly by looking at the front end.
+			await test.step( 'verify the saved product in frontend', async () => {
+				const permalink = await page
+					.locator( '.product-details-section__product-link a' )
+					.getAttribute( 'href' );
+
+				await page.goto( permalink );
+
+				// Verify product name
+				await expect(
+					page.getByRole( 'heading', {
+						name: productData.name,
+					} )
+				).toBeVisible();
+
+				// Verify price
+				await expect(
+					page.getByText( productData.productPrice ).first()
+				).toBeVisible();
+				await expect(
+					page.getByText( productData.salePrice ).first()
+				).toBeVisible();
+
+				// Verify summary
+				await expect(
+					page.getByText( productData.summary )
+				).toBeVisible();
+
+				// Verify description
+				await page.getByRole( 'tab', { name: 'Description' } ).click();
+
+				await expect(
+					page.getByText( productData.descriptionTitle )
+				).toBeVisible();
+				await expect(
+					page.getByText( productData.descriptionSimple )
+				).toBeVisible();
+				await expect(
+					page.getByText( productData.descriptionParagraph )
+				).toBeVisible();
+
+				// Verify inventory details
+				await expect(
+					page.getByText( `SKU: ${ productData.sku }` )
+				).toBeVisible();
+				// Note: GTIN is not displayed in the front end in the theme used in the test
+
+				// Note: Shipping class is not displayed in the front end in the theme used in the test
+
+				// Verify shipping dimensions
+				await page
+					.getByRole( 'tab', { name: 'Additional information' } )
+					.click();
+
+				await expect(
+					page.getByText( `Weight ${ productData.shipping.weight }` )
+				).toBeVisible();
+				await expect(
+					page.getByText(
+						`Dimensions ${ productData.shipping.length } × ${ productData.shipping.width } × ${ productData.shipping.height }`
+					)
+				).toBeVisible();
+			} );
+		} );
```

Comment:
This file is also updated and on `trunk` - [PR here](https://github.com/woocommerce/woocommerce/pull/54006/files#diff-b48bb915835e9e77dc0f38e1bd73b9c283ec373cd53af0139d51fb1a3b21de59)